stable/mysql/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+Getting Started:
```

Comment:
See my comments about the MariaDB NOTES.txt in #89, I think we can apply them here too :)